schema.sql

```diff
@@ -7,9 +7,43 @@ CREATE TABLE animals(
   escape_attempts    INT,
   neutered     boolean,
   weight_kg    REAL,
+  species_id INT references species(id),
+  owners_id INT references owners(id),
+  CONSTRAINT fk_species
+      FOREIGN KEY(species_id) 
+	  REFERENCES species(id)
+      ON DELETE SET NULL,
+  CONSTRAINT fk_owners
+      FOREIGN KEY(owners_id) 
+	  REFERENCES owners(id)
+      ON DELETE SET NULL,
   PRIMARY KEY(id)
 );
 
-ALTER TABLE animals
-ADD species varchar(255);
+CREATE TABLE owners(
+  id            INT GENERATED ALWAYS AS IDENTITY,
+  full_name    		VARCHAR(100),
+  age               INT,
+  PRIMARY KEY(id)
+);
 
+CREATE TABLE species(
+  id            INT GENERATED ALWAYS AS IDENTITY,
+  name    		VARCHAR(100),
+  PRIMARY KEY(id)
+);
+
+CREATE TABLE result(
+    id            INT GENERATED ALWAYS AS IDENTITY,
+    full_name    VARCHAR(100),
+    num          INT,
+    PRIMARY KEY(id)
+);
+
```

Comment:
- [OPTIONAL] I would advise you not to create a table because it isn't part of the requirement. I understand that you are using the table to store figures for one of the queries that you wrote, but there are more efficient ways to write the queries. 💻